CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ Every module contains its own CHANGELOG.md. Please refer to the module you are i
 * (crypto/keys) [#18026](https://github.com/cosmos/cosmos-sdk/pull/18026) Made public key generation constant time on `secp256k1`
 * (crypto | x/auth) [#14372](https://github.com/cosmos/cosmos-sdk/pull/18194) Key checks on signatures antehandle.
 * (types) [#18963](https://github.com/cosmos/cosmos-sdk/pull/18963) Swap out amino json encoding of `ABCIMessageLogs` for std lib json encoding
+* (x/gov) [#19383](https://github.com/cosmos/cosmos-sdk/pull/19383) Add Query for getting Last Proposal ID
```

Comment:
Could you move this in x/gov/changelog.md?

Reply:
Done in https://github.com/cosmos/cosmos-sdk/pull/19383/commits/9aa1406f4cdd60eb58a2b4871a372ebc2a2d95d2 !

Reply:
Thanks! Can you run `make lint-fix` too? https://github.com/cosmos/cosmos-sdk/actions/runs/7932961255/job/21660620155?pr=19383